### PR TITLE
Remove border bottom style from the download links

### PR DESCRIPTION
**Pull request by sentayhuJo**
🐿 v2.12.3
- Onsite message, ID `appPromoMobile` downloads links , `border bottom` style updated.

### CODE REVIEW

File: templates/partials/bottom/app-promo-mobile.html [hunk 1]
```diff
@@ -20,10 +20,10 @@
 		<!-- Button and link -->
 		<div class="n-messaging-banner__actions">
 			<div class="n-messaging-banner__action">
-				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” data-trackable=“onsite-message-link-iOS” target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
+				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” data-trackable=“onsite-message-link-iOS” style="border: 0;" target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
```
Comment (CourtneyLO) [inline templates/partials/bottom/app-promo-mobile.html]:
```suggestion
				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" data-n-messaging-banner-action="" data-trackable="onsite-message-link-iOS" target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
```

File: templates/partials/bottom/app-promo-mobile.html [hunk 2]
```diff
@@ -20,10 +20,10 @@
...
 			</div>
 			<div class="n-messaging-banner__action">
-				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” data-trackable=“onsite-message-link-googlePlay” target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
+				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” style="border: 0;" data-trackable=“onsite-message-link-googlePlay” target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
```
Comment (CourtneyLO) [inline templates/partials/bottom/app-promo-mobile.html]:
```suggestion
				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com" data-n-messaging-banner-action="" data-trackable="onsite-message-link-googlePlay" target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
```

File: templates/partials/bottom/app-promo-mobile.html [hunk 3]
```diff
@@ -20,10 +20,10 @@
 		<!-- Button and link -->
 		<div class="n-messaging-banner__actions">
 			<div class="n-messaging-banner__action">
-				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” data-trackable=“onsite-message-link-iOS” target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
+				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" class="n-messaging-banner__link" data-n-messaging-banner-action="" data-trackable="onsite-message-link-iOS" target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
```
Comment (CourtneyLO) [inline templates/partials/bottom/app-promo-mobile.html]:
```suggestion
				<a href="https://itunes.apple.com/app/apple-store/id1200842933?mt=8" data-n-messaging-banner-action="" data-trackable="onsite-message-link-iOS" target="_blank" role="link"><img src="https://www.ft.com/__assets/creatives/tour/apps/ios-download.svg" alt="Download from the iOS App store" height="35"></a>
```

File: templates/partials/bottom/app-promo-mobile.html [hunk 4]
```diff
@@ -20,10 +20,10 @@
...
 			</div>
 			<div class="n-messaging-banner__action">
-				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com" class=“n-messaging-banner__link” data-n-messaging-banner-action=“” data-trackable=“onsite-message-link-googlePlay” target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
+				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com" class="n-messaging-banner__link" data-n-messaging-banner-action="" data-trackable="onsite-message-link-googlePlay" target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
```
Comment (CourtneyLO) [inline templates/partials/bottom/app-promo-mobile.html]:
```suggestion
				<a href=" https://play.google.com/store/apps/details?id=com.ft.news&referrer=utm_source%3Dhelp.ft.com"  data-n-messaging-banner-action="" data-trackable="onsite-message-link-googlePlay" target="_blank" role="link"><img src="https://www.ft.com/__origami/service/image/v2/images/raw/https%253A%252F%252Fwww.ft.com%252F__assets%252Fcreatives%252Ftour%252Fapps%252Fgoogle-play-badge-3x.png?source=ip-envoy" height="35" alt="Download on Google Play"></a>
```

File: templates/partials/bottom/app-promo-mobile.html (hunk 1 above)
Comment (CourtneyLO) [reply templates/partials/bottom/app-promo-mobile.html]: I don't believe you need this styling here.

File: templates/partials/bottom/app-promo-mobile.html (hunk 2 above)
Comment (CourtneyLO) [reply templates/partials/bottom/app-promo-mobile.html]: Same as above

File: templates/partials/bottom/app-promo-mobile.html (hunk 4 above)
Comment (sentayhuJo) [reply templates/partials/bottom/app-promo-mobile.html]: Done, already test it , and its clear now, thank you

**Pull request merged**